lib/diagnostic.js

```diff
@@ -47,7 +47,8 @@ let scopes = [/* your response here */];
 // executing it.
 //
 let fireOnAlderaan = function () {
-  // your code here
+  deathStar.fire().bind(alderaan);
+  return deathStar.fire();
 };
```

Comment:
`bind` binds the context, but does not run the function. So when you tried to `return deathStar.fire()`, the error says "Cannot read property of 'undefined'." 

`.call` is the method you want to use here:
```js
let fireOnAlderaan = function () {
  return deathStar.fire.call(alderaan);
};
```